ansible/group_vars/all/config.yml

```
@@ -154,6 +154,20 @@ configuration_all:
     server: true
     client: false
     nuxtclient: false
+  FILES_STORAGE__S3_ENDPOINT:
+    value: "https://storage-{{ DOMAIN }}"
+    server: true
+    client: false
+    nuxtclient: false
+  FILES_STORAGE__S3_BUCKET:
+    value: "{{ DOMAIN }}"
+    server: true
+    client: false
+    nuxtclient: false
+  FILES_STORAGE__S3_REGION:
+    server: true
+    client: false
+    nuxtclient: false
```

pleasse add the default vaule "eu-central-1"
```suggestion
  FILES_STORAGE__S3_REGION:
    value: "eu-central-1"
    server: true
    client: false
    nuxtclient: false
```